elasticsearch-model/lib/elasticsearch/model/adapters/mongoid.rb

```diff
@@ -66,17 +66,26 @@ def __find_in_batches(options={}, &block)
             options[:batch_size] ||= 1_000
             items = []
 
-            all.each do |item|
+            query = options.delete(:query)
+            named_scope = options.delete(:scope)
+            preprocess = options.delete(:preprocess)
+            scope = scoped
```

Comment:
What does `scoped` in Mongoid do? Wouldn't this respect any "default" scope? I don't wanna do that, it's best that we default to "all records" during import, and make people use the scope, query options to filter the records to import.